examples/features/keepalive/README.md

```diff
@@ -0,0 +1,15 @@
+# Keepalive
+
+This example illustrates how to set up client-side keepalive pings and
+server-side keepalive ping enforcement and connection idleness settings.  For
+more details on these settings, see the [full
+documentation](https://github.com/grpc/grpc-go/tree/master/Documentation/keepalive.md).
+
+
+```
+go run server/main.go
```

Comment:
`GODEBUG=http2debug=2` for server too?

Reply:
Considered and decided against since the client shows all the frames already.  No reason to show it twice.